### ISSUE TITLE
Usage of classes without declared namespaces not detected

### ISSUE THREAD

**Issue by elazar** [Insolita/unused-scanner#2]
One of the dependencies of my project is `datadog/php-datadogstatsd`. This library uses a [classmap autoloader](https://getcomposer.org/doc/04-schema.md#classmap) and references a directory containing a single class file that does not include a `namespace` directive.

Under one of the directories included in the `scanDirectories` setting, I have a class that extends from this library's class. This class usage is not detected.

**Comment by Insolita** [comment]
Not clear. I just install this library, and last version  use psr4 autoload,  With example 
```
<?php

namespace Acme\Scanner;

use DataDog\BatchedDogStatsd;

class DatadogUsage extends BatchedDogStatsd
{
    public function foo(){
        echo 'bar';
    }
}
```
This usage detect successfully

Show your example

**Comment by elazar** [comment]
Looks like the namespace was added in version 1.0.0. Try an older version. I'm using 0.3.0.

**Comment by Insolita** [comment]
try with updated version

**Comment by elazar** [comment]
Appears to be fixed under 1.0.6. Thanks for the prompt reply! 😄
